.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install bats
```

```suggestion
          sudo apt-get install -y --no-install-recommends bats
```
Even though the GH-Actions runners are well configured, making the "non interactive" explicit is always a good help for future contributors, WDYT?

---

OK